src/writer/sync.py

```
@@ -248,36 +248,48 @@ def process_batched_operations(self) -> None:
                 
                 # Process file updates
                 if self.operation_queues.files_to_update:
                     self.log(f"Updating {len(self.operation_queues.files_to_update)} files")
                     for dest_path, src_path in self.operation_queues.files_to_update.copy().items():
                         if self.should_copy(src_path, dest_path):
-                            self.copy_file(src_path, dest_path)
-                            self.log("File updated:", self.rel(src_path))
+                            try:
+                                self.log("File update:", self.rel(src_path))
+                                self.copy_file(src_path, dest_path)
+                            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)